libkbfs/errors.go

```diff
@@ -169,8 +169,12 @@ type ReadAccessError struct {
 
 // Error implements the error interface for ReadAccessError
 func (e ReadAccessError) Error() string {
+	t := tlf.Private
+	if e.Public {
+		t = tlf.Public
+	}
 	return fmt.Sprintf("%s does not have read access to directory %s",
-		e.User, buildCanonicalPathForTlfName(e.Public, e.Tlf))
+		e.User, buildCanonicalPathForTlfName(t, e.Tlf))
```

Comment:
What if it's a team folder? I guess this is fine if we decide to let team folders stay in `/keybase/private`